doc/OpenMS_tutorial/OpenMS_Tutorial.doxygen

```diff
@@ -608,7 +608,7 @@ Now that we assembled the set of all protein accessions we are ready to compare
 <b>Example: Tutorial_Functionality.cpp</b>
 <br>
 Add method functionality
-\snippet Tutorial_Functionality.cpp Functionality_2
+\snippet Tutorial_Final.cpp Functionality_2
```

Comment:
Copy paste error?

Reply:
This should be Ok, take multiple snippets from the Tutorial_Final.cpp - The Example: Should be Tutorial_Final.cpp.

Reply:
Hm yes. I just want that readers easily find the right file to look up and run the snippet.
Btw: I think you added a "y" too much.